Sources/MapboxMaps/Snapshot/Snapshotter.swift

```diff
@@ -266,3 +266,21 @@ extension Snapshotter: MapEventsObservable {
         return handler
     }
 }
+
+// MARK: - Clear data
+
+extension Snapshotter {
```

Comment:
Same as MapboxMap, on Android, we added clearData as a function inside the `Snapshotter` instead of using extension function, I think it could just be an implementation detail.

Reply:
It's just an implementation detail and won't impact public API.